free-3.md

```diff
@@ -10,41 +10,108 @@ _What need is met by your idea? Who are you helping? What is that person's
+with the tasks and gives the user a visual output that outlines the schedule.
+
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+The idea excites me because I know that nearly all of my friends in college have
+struggled with procrastination. I don't think people need to or should be productive
+all the time, but when you really need motivation to be productive, it's nice to
+have a really clear plan of action. Coming up with this action plan is difficult
+and time-consuming to do on your own, so having a language to express the tasks you
+need to accomplish and having a schedule computed for you could be an effective tool.
+I came up with this idea after realizing how much time I spend adding things to my
+task manager and then scheduling tasks in my calendar which sort of negates the
+sort of productivity I'm trying to achieve.
+
 
 ### Domain
 _Describe the project's domain in five words._
 
+Task organization, automatic schedule generation
+
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+The user would input their tasks in a simple text file. They would create
+task objects that could have several parameters including predicted length,
+priority, deadline, and frequency. The user could then group the tasks together
+into different categories of tasks, such as homework, jobs, self-care, etc. The
+idea is to have the language feel like the user is sort of writing down notes
+about the things they need to do, and leave the organization and scheduling to
+the computer.
+
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+The computer will associate a certain number to each task, and this number 
+reflects the overall priority of the task. If the user specifies that a certain
+task is a priority, it will automatically be associated with a lower number. Based
+on the numbered order of all the tasks, the task will be inserted into a certain 
+time block on a daily calendar. If the user connects a calendar app with the program,
+the output will be reflected by filled in slots on the calendar. Otherwise, an excel file
+will be generated with a table of each hour of the day and its associated task. 
+Errors in syntax or overloading parameters could happen, and these errors would be 
+communicated at compile time and hopefully highlighted in the code.
+
 
 ### Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+It should be easy to generate task schedules with this language. It would
+be difficult to plan out a very specific schedule with this due to the randomness
```

Comment:
So a user couldn't manually override if they wanted to do something in a different order?